permits/serializers.py

```diff
@@ -69,7 +69,19 @@ class Meta:
 
 class WotPropertiesValuesSerializer(serializers.RelatedField):
     def to_representation(self, value):
-        wot_properties = services.get_wot_properties(value, value_with_type=True)
+        current_user = None
+        request = self.context.get("request", None)
+        if request:
+            current_user = request.user
+            basic_authentication = request.session._SessionBase__session_key
```

Comment:
Basic auth is disabled by default.  Looks like you mean session_authentication ?